tests/test_masks.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+
+from src.masks import get_mask_account, get_mask_card_number
+
+
+# Тесты для get_mask_card_number
+@pytest.mark.parametrize("card_number, expected", [
+    ("1234567890123456", "1234 56** **** 3456"),
+    ("1234567812345678", "1234 56** **** 5678"),
+    ("123456789012", "1234 56** **** 9012"),
+    ("12345678901234567890", "1234 56** **** 7890"),
+    ("123456", "1234 56** **** 3456"),
+    ("0000000000000000", "0000 00** **** 0000"),
+    ("9999999999999999", "9999 99** **** 9999"),
+])
+def test_get_mask_card_number_valid(card_number, expected):
+    assert get_mask_card_number(card_number) == expected
+
+
+@pytest.mark.parametrize("invalid_card, expected_error", [
+    ("", "Номер карты не может быть пустым"),
+    ("12345", "Номер карты должен содержать минимум 6 цифр"),
+    ("1234abcd56789012", "Номер карты должен содержать только цифры"),
+    ("1234 5678 9012 3456", 'Номер карты должен содержать только цифры'),
+    ("1234-5678-9012-3456", "Номер карты должен содержать только цифры"),
+    (None, "Номер карты должен быть строкой"),
+    (True, "Номер карты должен быть строкой"),
+    ([], "Номер карты должен быть строкой"),
+    ({}, "Номер карты должен быть строкой"),
+    (3.14, "Номер карты должен быть строкой"),
+])
+def test_get_mask_card_number_invalid(invalid_card, expected_error):
+    with pytest.raises((ValueError, TypeError)) as exc_info:
```

Comment:
Правильнее проверять так:
```suggestion
    with pytest.raises((ValueError, TypeError), match=excepted_error):
```